openshift/release/create-release-branch.sh

```diff
@@ -30,5 +30,6 @@ if [ -d "openshift/patches-${release}" ]; then
 fi
 git apply $PATCH_DIR/*
 make RELEASE=$release generate-release
+make RELEASE=ci generate-release
```

Comment:
I guess that simply adds the label "1.3.0" as done manually here #1136 and is also missing from https://github.com/openshift/knative-serving/blob/release-v1.4/openshift/release/knative-serving-ci.yaml correct. 
Over time `make generate-release` was minimized to simply update labels it seems: https://github.com/openshift/knative-serving/blob/main/openshift/release/resolve.sh#L21-L23 (the comment there is wrong obviously).